prepend to $PATH rather than append, to avoid picking up system compiler (g++)
author: Kenneth Hoste (HPC-UGent)
see also https://bitbucket.org/berkeleylab/metabat/issues/52
--- berkeleylab-metabat-37db58fe3fda/SConstruct.orig	2017-09-01 07:28:02.000000000 +0200
+++ berkeleylab-metabat-37db58fe3fda/SConstruct	2019-02-26 16:13:03.150132631 +0100
@@ -7,7 +7,7 @@
 def ENV_update(tgt_ENV, src_ENV):
     for K in src_ENV.keys():
         if K in tgt_ENV.keys() and K in [ 'PATH', 'LD_LIBRARY_PATH', 'LIB', 'INCLUDE' ]:
-            tgt_ENV[K]=SCons.Util.AppendPath(tgt_ENV[K], src_ENV[K])
+            tgt_ENV[K]=SCons.Util.PrependPath(tgt_ENV[K], src_ENV[K])
         else:
             tgt_ENV[K]=src_ENV[K]
 
